src/main/resources/sql/data.sql

```diff
@@ -1,5 +1,5 @@
 -- Add several users
-
+INSERT INTO `user` (`login`, `email`, `password`, `first_name`, `last_name`, `profile_photo`, `created_at`) VALUES ('test', 'test@gmail.com', 'test', 'test', 'test', 'https://cdn.pixabay.com/photo/2016/08/08/09/17/avatar-1577909__340.png' ,'2008-03-01 17:05:33');
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([plsql:S1192](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=plsql%3AS1192))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=134&issues=AWVbiOH6ZiUEgfxFxpi7&open=AWVbiOH6ZiUEgfxFxpi7)

[ISSUE_KEY]: AWVbiOH6ZiUEgfxFxpi7